src/Dapr.Actors/Runtime/ActorTypeInformation.cs

```diff
@@ -23,6 +23,13 @@ public ActorTypeInformation()
         {
         }
 
+        /// <summary>
+        /// Gets the name of the actor type represented by the actor.
+        /// </summary>
+        /// <value>The <see cref="string"/> name of the actor type represented by the actor.</value>
+        /// <remarks>Defaults to the name of the class implementing the actor.</remarks>
```

Comment:
Could you also add the override behavior of type name using attribute here as well in remarks.

Reply:
Updated.